larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -556,13 +609,23 @@ TfLiteRegistration* Register_BCONV_2D64_OPT() {
 // Use this registration wrapper to decide which impl. to use.
 TfLiteRegistration* Register_BCONV_2D() {
 #if defined TFLITE_WITH_RUY
+
 #if RUY_PLATFORM(ARM_32)
   return Register_BCONV_2D32_OPT();
 #else  // ARM 64 and x86
   return Register_BCONV_2D64_OPT();
 #endif
+
 #else  // disabled TFLITE_WITH_RUY
-  static_assert(false, "LCE needs to be compiled with TFLite RUY activated.");
+
+#if RUY_PLATFORM(ARM_32)
+  return Register_BCONV_2D32_REF();
+#else  // ARM 64 and x86
+  static_assert(false,
+                "LCE do not support 64-bit binary convolution kernel "
```

Comment:
```suggestion
                "LCE does not support 64-bit binary convolution kernel "
```

Reply:
What do you think about always running the `32_REF` registration here, also on 64-bit systems?